packages/aws-cdk-lib/aws-elasticloadbalancingv2/test/nlb/load-balancer.test.ts

```diff
@@ -800,34 +827,32 @@ describe('tests', () => {
 
     test('can look up security groups', () => {
       // GIVEN
-      const stack = new cdk.Stack();
-      const vpc = new ec2.Vpc(stack, 'Stack');
-      const sg = new ec2.SecurityGroup(stack, 'SG', { vpc });
-
-      // WHEN
-      const nlb = new elbv2.NetworkLoadBalancer(stack, 'LB', {
-        vpc,
-        internetFacing: true,
-        securityGroups: [sg],
+      const app = new cdk.App();
+      const stack = new cdk.Stack(app, 'Stack', {
+        env: {
+          account: '123456789012',
+          region: 'us-west-2',
+        },
       });
 
-      // THEN
-      expect(nlb.securityGroups).toEqual([`${sg.securityGroupId}`]);
-    });
-
-    test('can look up with no security groups', () => {
```

Comment:
This test was removed because it was not a test as the test title suggests, and it was a duplicate test where NLB was initialized without a security group.

Reply:
Added similar test.
https://github.com/aws/aws-cdk/pull/28494/files#diff-8337948b99d70f9c9a158c9b6c7dc15a6061a239d6d2fcc8f697b05f56ed0868R763